docs/_docs/analysis.md

```diff
@@ -156,10 +157,41 @@ form factor of unity.
 `dq`        | _q_ spacing (1/Å)
 `com=true`  | Treat molecular mass centers as single point scatterers
 
+### Gyration Tensor
 
-### Radius of Gyration
+This calculates the gyration eigenvalues for a group of particles
```

Comment:
Is it mass-weighted?

Reply:
No, the mass-weighted equivalent is the inertia tensor.

Reply:
Perhaps it could be an option?